qa/rh/downstream.yaml

```diff
@@ -37,13 +37,13 @@ pkgs:
     - python-cephfs
     - rbd-mirror
 
-repos:
-  rhel_7:
+rhel_repos:
+  '7':
     - rhel-7-server-rpms
     - rhel-7-server-optional-rpms
```

Comment:
@rakeshgm as per docs we dont need optional-rpms ( remember some time ago we faced some dependency error in octo lab because of having optional rpms enabled by default) better to avoid i guess

Reply:
correct, but we install many packages based on the tests and some might be from optional-rpms. these packages are from `install` task.

Reply:
ohh okay..ceph cluster install task wont fail atleast.

If there is any package which we are picking specifically for running io or anyother outside of ceph from optional-rpms  then it can be usefull. i couldnt find any here https://github.com/ceph/teuthology/search?q=extra_system_packages&unscoped_q=extra_system_packages  https://github.com/ceph/teuthology/blob/rh/teuthology/task/install/packages.yaml,  are we installing any other packages from any other tasks dependent on optional-rpms?

i have a list of all available packages from optional-rpms here : http://magna002.ceph.redhat.com/ceph-qe-logs/subhash/misc/optional-rpms-packages , may be can check here if any are useful in teuthology ( to make sure these dependencies are not enabled while we deploy ceph)

Reply:
we some times install packages like this https://github.com/red-hat-storage/ceph/blob/rh-luminous/qa/suites/rgw/singleton/all/radosgw-admin.yaml#L9 . some of the packages might come from optional. I just thought its good to have. no harm I believe.